lib/ansible/modules/windows/win_service.ps1

```diff
@@ -144,6 +214,21 @@ Function Get-DelayedStatus($name) {
     $delayed
 }
 
+Function Get-RecoveryActions($name) {
+  $recovery_actions_key = "HKLM:\System\CurrentControlSet\Services\$name"
+  try {
+      $recovery_actions = ""
+      $bytes = (Get-ItemProperty -LiteralPath $recovery_actions_key).FailureActions
+      $bytes | ForEach-Object { $recovery_actions+=('{0:x2}' -f $_).ToString() }
+      $recovery_actions = $recovery_actions.substring(0,$recovery_actions.length)
+  } catch {
+      $recovery_actions = $false
+  }
+
+  $recovery_actions
```

Comment:
Please use return otherwise it might cause issues if piped since it outs as output